site/src/patterns/card.md

```diff
@@ -48,28 +48,79 @@ This card type contains a super title labelled by the card's type, a title, an i
 
 ```html
 <article class="card position-relative">
-	<div class="card-content">
-		<div class="media">
+	<div class="card-content display-flex">
+		<div class="media flex-grow-1 align-items-stretch">
 			<div class="media-left">
 				<div class="image image-64x64">
 					<img src="~/src/scaffold/media/video-media.png" />
 				</div>
 			</div>
+			<div class="media-content display-flex flex-direction-column align-items-space-between">
+				<div>
+					<p class="text-transform-uppercase margin-bottom-xxs">Videos</p>
+					<p class="font-size-lg font-weight-semibold margin-bottom-xxs">
+						Media content is a place to put your text content.
+					</p>
+					<p class="color-text-subtle margin-bottom-xs">Videos to help you get started.</p>
+				</div>
+				<a class="text-decoration-none justify-self-stretch margin-top-auto" href="#"
+					>Watch videos</a
+				>
+			</div>
+		</div>
+	</div>
+</article>
+```
+
+## Summary card
+
+These cards use standard `.card` subcomponents, border atomics to create a left border, and (because they have only one clickable item inside) the [stretched-link component](../components/stretched-link.md). They use `.card-content-description` to truncate a string of summary text. Note that these will typically not be full-width elements, but will be displayed side-by-side with other cards of this type.
+
+```html
+<article class="card position-relative border-left-lg border-left-color-accent">
+	<div class="card-content">
+		<a href="#" class="card-title stretched-link">The title of the card</a>
+		<p class="card-content-description">
+			The longer description of the card, which will eventually become truncated the more we repeat
+			this sentence. The longer description of the card, which will eventually become truncated the
+			more we repeat this sentence. The longer description of the card, which will eventually become
+			truncated the more we repeat this sentence. The longer description of the card, which will
+			eventually become truncated the more we repeat this sentence.
+		</p>
+	</div>
+</article>
+```
+
+## Testimonial card
+
+The testimonial card provides a place for a user's opinion to be showcased. Notable modifications from Icon Card include using `border-radius-rounded` on the image, as well as slightly different font and text sizes. There are also several flexbox based atomics applied to ensure links justify at the ends of cards when placed in a grid.
+
+```html
+<article class="card">
+	<div class="card-content font-size-lg display-flex flex-direction-column">
```

Comment:
Looks good in grid 🙌🏻

![image](https://user-images.githubusercontent.com/57374379/186522964-b12347f0-53d5-4457-9f20-73941c236583.png)

Reply:
What about in mobile? 
![image](https://user-images.githubusercontent.com/94572161/186523746-5b5bd9e6-9240-4a1a-bacc-a18f3a7b47fd.png)

Reply:
In mobile cards would stack one on top of the other ;) 

![image](https://user-images.githubusercontent.com/57374379/186525040-d9c4dd00-2451-4a03-a501-6def4d434fbb.png)

Reply:
Nevermind, it has to do with the example markup. Ignore my comment please

Reply:
Yes, ideally the card doesn't change based on screensize, but the grid certainly would.